src/app.ts

```diff
@@ -29,4 +42,5 @@ app.listen(3000, () => {
     console.log('Server started on port 3000');
 });
 
-app.get('/', getAllDatabases);
\ No newline at end of file
+app.get('/homepage' , getHomePage)
+app.get('/jobRoles', getAllJobRoles);
```

Comment:
by adding all of the job role stuff in this branch we can't progress the branding ticket until the job roles ticket is complete. I recommend removing all the job roles stuff and just make this PR about the branding